### PR TITLE
Added unit test cases for signin and signup component

### PR DESCRIPTION
**Pull request by Utkarsh-81**
Added unit test cases for signin and signup component

### CODE REVIEW

File: __test__/signin.test.jsx [hunk 1]
```diff
@@ -1,16 +1,85 @@
...
+    expect(buttonElement).toHaveLength(1);
+  });
+
+  it("Intially email input should be empty", () => {
+    const emailInput = screen.getByPlaceholderText("Email Address");
+    expect(emailInput.value).toBe("");
+  });
+
+  it("Intially password input should be empty", () => {
+    const passwordInput = screen.getByPlaceholderText("Email Address");
```
Comment (pratheesh-repo) [inline __test__/signin.test.jsx]: screen.getByPlaceholderText("Password");

File: __test__/signin.test.jsx [hunk 2]
```diff
@@ -1,16 +1,85 @@
...
+  it("password input should change", () => {
+    const passwordInput = screen.getByPlaceholderText("Password");
+
+    const testValue = "Akash@63517";
+    fireEvent.change(passwordInput, { target: { value: testValue } });
+
+    expect(passwordInput.value).toBe(testValue);
+  });
+
+  it("button should not be disabled when inputs exists", () => {
```
Comment (pratheesh-repo) [inline __test__/signin.test.jsx]: Signin Button

File: __test__/signup.test.jsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    expect(emailInput).toHaveAttribute("placeholder", "Email Address");
+
+    const passwordInput = screen.getByPlaceholderText("Password");
+    expect(passwordInput).toHaveAttribute("placeholder", "Password");
+
+    const countryInput = screen.getByPlaceholderText("Select Country");
+    expect(countryInput).toHaveAttribute("placeholder", "Select Country");
+  });
+
+  it("button input should be rendered", () => {
```
Comment (pratheesh-repo) [inline __test__/signup.test.jsx]: Signup button

File: __test__/signup.test.jsx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    expect(email.value).toBe("");
+
+    const password = screen.getByPlaceholderText("Password");
+    expect(password.value).toBe("");
+
+    const country = screen.getByPlaceholderText("Select Country");
+    expect(country.value).toBe("");
+  });
+
+  it("button should be disabled", () => {
```
Comment (pratheesh-repo) [inline __test__/signup.test.jsx]: SignUp

File: __test__/signin.test.jsx (hunk 1 above)
Comment (Utkarsh-81) [reply __test__/signin.test.jsx]: Addressed

File: __test__/signin.test.jsx (hunk 2 above)
Comment (Utkarsh-81) [reply __test__/signin.test.jsx]: Addressed

File: __test__/signup.test.jsx (hunk 3 above)
Comment (Utkarsh-81) [reply __test__/signup.test.jsx]: Addressed

File: __test__/signup.test.jsx (hunk 4 above)
Comment (Utkarsh-81) [reply __test__/signup.test.jsx]: Addressed

**Pull request merged**